File: ase/misc/clr/assembly.js

var EXE = CLR.GetExecutingAssembly();
var Core = CLR.GetCore();
print(Core.GetType());
var Forms = CLR.Load("System.Windows.Forms");
var asname = Core.GetName();
print(asname.GetType());
// var kp = asname.KeyPair();
// print(kp.GetClass());
// print(kp.PublicKey());
// var Core.GetClass().Load

print(EXE.FullName);
print(Core.FullName);
// print(Forms.FullName());

File: ase/misc/clr/assembly.lua

local EXE = CLR.GetExecutingAssembly();
local Core = CLR.GetCore();
print("EA: " .. EXE.FullName);
print("Core: " .. Core.FullName);
print("CORE Location: " .. Core.Location);

local Forms = CLR.Load("System.Windows.Forms");
print(Forms.Location);

local asname = Core.GetName();
print(asname.GetType());
local kp = asname.KeyPair;
if kp == null then
  print("kp is null");
end
-- print(kp.PublicKey());

-- var Forms = CLR.Load("System.Windows.Forms");
-- print(Forms.FullName());

